modules/programs/pimsync/default.nix

```diff
@@ -112,6 +112,15 @@
       ];
 
       accountSettings = calendarConfig ++ contactConfig;
+
+      localStorageDir = name: acc: lib.attrsets.getAttrFromPath [ "local" "path" ] acc;
+
+      calendarLocalStorageDirs = lib.mapAttrsToList localStorageDir calendarAccounts;
+      contactLocalStorageDirs = lib.mapAttrsToList localStorageDir contactAccounts;
+      localStorageDirs = calendarLocalStorageDirs ++ contactLocalStorageDirs;
+
+      mkTmpFileRule = (dir: ''d ${dir} 0755 ${config.home.username} ${config.home.username} - -'');
```

Comment:
`0755` seems a bit wide, I dont want "all" to be able to read my calendars :o  Can you check if there is a way to set it to default umask or otherwise 0750 ?